_templates/how-to.md

```diff
@@ -1,26 +1,72 @@
 ---
 title: [Character Limit: 255, Recommendation is to have <= 75 characters for a title as this can impact indexing by search engines like Google, Bing, etc. The article title should be a combination of the main problem statement and if applicable, the key environment statement. Also determines the title in the table of contents.]
-description: [Brief description of the How-To article]
-type: [HowTo]
-page_title: [Text that will populate the HTML file’s head:title attribute]
+description: [Character Limit: 255, Recommendation is to have <= 155 characters for a description as this can impact indexing by search engines like Google, Bing, etc. The description should be brief overview of the how-to article. It should exclude unnecessary phrases like "This article contains ..."]
```

Comment:
Add this as a detail when explaining for the character recommendation. E.g.

Character Limit: 255, Recommended length: <= 155 characters. The description is what search engines usually show in the search results. Typically, search engines truncate the description after the 155th character. The description should be a brief overview of the how-to article. It should exclude unnecessary phrases like "This article contains ..."